pkg/controller/direct/monitoring/monitoringdashboard_controller.go

```diff
@@ -236,6 +236,7 @@ func (a *dashboardAdapter) Update(ctx context.Context, u *unstructured.Unstructu
 			Dashboard: a.desired,
 		}
 		req.Dashboard.Name = a.fullyQualifiedName()
+		req.Dashboard.Etag = a.actual.Etag
```

Comment:
Maybe we can let the base direct controller to handle etag (`status.etag` or what ever we call it, not  the `status.observedState.etag`)